buildSrc/subprojects/performance-testing/src/main/groovy/gradlebuild/performance/tasks/PerformanceTest.groovy

```diff
@@ -227,6 +271,7 @@ abstract class PerformanceTest extends DistributionTest {
 
         private void addExecutionParameters(List<String> result) {
             addSystemPropertyIfExist(result, "org.gradle.performance.scenarios", scenarios)
+            addSystemPropertyIfExist(result, "org.gradle.performance.testProject", getTestProjectName().getOrNull())
```

Comment:
Can this ever be `null`?

Reply:
It is currently `null` for all the old performance test tasks. When we are finished migrating to the new infrastructure, it should never be null.